ext/standard/var_unserializer.re

```diff
@@ -71,6 +77,17 @@ PHPAPI void var_push_dtor(php_unserialize_data_t *var_hashx, zval *rval)
 	ZVAL_COPY(tmp_var, rval);
 }
 
+/* Create a copy of rval (of type IS_OBJECT, with method "__wakeup" defined). Defer the call to __wakeup. */
+PHPAPI void var_push_wakeup(php_unserialize_data_t *var_hashx, zval *rval)
+{
+	/* TODO: This could be refactored into a common macro/data structures with var_tmp_var once the implementation is certain. */
+	zval *tmp_var = var_tmp_wakeup_var(var_hashx);
+    if (!tmp_var) {
```

Comment:
Correct, it was mixed in this file, so I wasn't sure. Will fix https://github.com/php/php-src/blob/master/CODING_STANDARDS

> When indenting, use the tab character.